sdk/firstload/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "highlight.run",
-	"version": "9.11.1",
+	"version": "9.12.0",
```

Comment:
doesn't look like this was done with `yarn changeset version`. why did we need to do this manually?

Reply:
I reran with that. I think I just ran `yarn changeset` before